config/features.yml

```diff
@@ -336,6 +336,10 @@ features:
   va_global_downtime_notification:
     actor_type: user
     description: Enables global downtime notification- do not use in production
+  va_online_mobile_specific_menu:
```

Comment:
Could we change the name to something like `mega_menu_mobile` or `mega_menu_mobile_v2`?